http-svr/xmlrpc_client.ml

```diff
@@ -206,6 +206,8 @@ let with_reusable_stunnel ?use_fork_exec_helper ?write_to_log ?verify_cert host
         StunnelDebug.error "get_reusable_stunnel: failed to acquire a working stunnel to connect to %s:%d" host port;
         raise Stunnel_connection_failed
   in loop ()
+(* supress warning about unused variables for the verify_cert parameter *)
+[@@ocaml.warning "-27"]
```

Comment:
Are you sure your branch is up to date? For me it shows this and the parameter is used:
```
     has been checked out and guaranteed to work. *)
  let with_reusable_stunnel ?use_fork_exec_helper ?write_to_log ?verify_cert host port f =
    (* 1. First check if there is a suitable stunnel in the cache. *)
>>  let verify_cert = Stunnel.must_verify_cert verify_cert in
```